generate-examples/generate-cert-chain.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# this script generates a 3-cert ECDSA P-256 chain (root -> CA -> issuer) valid for 5 years
+
+# directory where intermediate files are kept
+tmpdir=certs
+mkdir -p $tmpdir
+# directory where generated JWK files are stored
+outdir=certs
+mkdir -p $outdir
+
+# generate self-signed root CA cert
+openssl req -x509 -new -newkey ec:<(openssl ecparam -name secp521r1) -keyout $tmpdir/root_CA.key -out $tmpdir/root_CA.crt -nodes -subj "/CN=SMART Health Card Example Root CA" -days 1825
```

Comment:
Re: `secp521r1` here: obviously at the level of our spec we want to leave algorithm choices as trust-framework-specific decisions, but for the sake of our examples is it feasible to use P-256 across the board?

Reply:
We can discuss this, but a root CA typically uses stronger security params.

Reply:
Agreed with the assumption here, which is: implementers will need to support specific algorithms that a given trust framework mandates. Given this, my preference would be to keep our examples as "decision free" as they can be. But will defer to you on this.

Reply:
When all the code is ready and I flip the "ready for review" switch, we can ask for larger feedback; it will be easy to change these params and generate the examples we want.